testexamples/ethereum_block_handler_test.go

```diff
@@ -13,7 +13,13 @@ import (
 // destination table and custom table.
 func FindBlockHandlerString(blockNumber string, deps *utils.Deps) (bool, error) {
 	if blockNumber == "2" {
-		deps.DestinationDB.Create(&ethereum.Block{Number: 2})
+		// Exercise the code to read source logs table from the handler.
+		var logs []*ethereum.Log
+		deps.SourceDB.Table("ethereum.logs").Where("block_number = ?", 2).Find(&logs)
```

Comment:
"ethereum_mainnet.logs" or "ethereum_holesky.logs"?

Reply:
what schema/table is available totally depends on the caller provided `EthereumData`. In this example I had the same data in all three schemas ethereum, ethereum_mainnet, ethereum_holesky so they all work, but I can update this one to be ethereum_holesky for better illustration.